lib/Varien/Simplexml/Element.php

```diff
@@ -453,7 +453,7 @@ public function setNode($path, $value, $overwrite=true)
         foreach ($arr1 as $v) {
             if (!empty($v)) $arr[] = $v;
         }
-        $last = sizeof($arr)-1;
+        $last = count($arr)-1;
```

Comment:
since you spaced out the other "ifs" maybe we can make this one `count($arr) - 1;`? sorry for the nitpicky comment :-)

Reply:
fixed